Tests/TuistSupportTests/Utils/FileHandlerTests.swift

```diff
@@ -40,15 +41,18 @@ final class FileHandlerTests: XCTestCase {
 
     func test_replace_cleans_up_temp() throws {
         // Given
-        let tempFile = try TemporaryFile()
-        let destFile = try TemporaryFile()
-        let count = try countItemsInRootTempDirectory(appropriateFor: destFile.path.asURL)
+        let temporaryPath = try self.temporaryPath()
```

Comment:
I had to change this because it was causing the builds to [fail](﻿﻿https://github.com/tuist/tuist/pull/656/checks?check_run_id=297949736#step:5:6394)